lib/Nusoap/nusoap.php

```diff
@@ -4242,7 +4242,7 @@ function send_response() {
 		foreach($this->outgoing_headers as $hdr){
 			header($hdr, false);
 		}
-		print $payload;
+		print esc_html( $payload );
```

Comment:
Check if content needs to get escaped.

Reply:
Switched to wp_kses with call to Util_Ui::get_allowed_html_for_wp_kses_from_content for allowed HTML tags